config/locales/en/activerecord.yml

```diff
@@ -11,6 +11,11 @@ en:
 
     errors:
       models:
+        mentor:
```

Comment:
You might need to add this to the active model locales too if this goes in https://github.com/DFE-Digital/itt-mentor-services/pull/278

Reply:
Thanks for the heads up. I'll add it if it becomes necessary.